packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/RetroTopic.tsx

```diff
@@ -84,6 +143,23 @@ const RetroTopic = (props: Props) => {
           </AnchorIfEmail>
         </td>
       </tr>
+      {showSummary && (
+        <tr>
+          <td align='left' style={{lineHeight: '22px', fontSize: 14}}>
+            <tr>
+              <td style={topicTitleStyle}>{'Topic Summary:'}</td>
+            </tr>
+            <tr>
+              <td
+                style={subtitleStyle}
+              >{`AI generated summaries are a premium feature. We'll share them with you in your first few retros so you can see what they're like.`}</td>
```

Comment:
-1 I don't see any check for the tier, so the message could be misleading for team/enterprise users.